test/suites/wdio.conf.js

```diff
@@ -139,17 +139,17 @@ exports.config = {
 	 * polls the wikibase docker container for installed extensions
 	 */
 	before: async () => {
-		await WikibaseApi.initialize();
+		await WikibaseApiPatch.initialize();
 		defaultFunctions.init();
 
-		if ( !browser.config.installed_extensions ) {
+		if ( !browser.options.installed_extensions ) {
```

Comment:
Let's add a TODO here that this is a hack and should be changed ASAP. As I understand WDIOs behaviour, it is not encuraged to use this object for config/options (anymore).